testing/Overview.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {mount, shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ItemDescription from '../client/src/Components/itemDescription.jsx';
+import ShippingInformation from '../client/src/Components/ShippingInformation.jsx'
+import globalFetch from 'jest-fetch-mock';
+import sinon from 'sinon'
+import Overview from '../client/src/Components/Overview.jsx'
+import Cart from '../client/src/AddToCart.jsx'
+import ReactDOM from 'react-dom'
+import ReturnPolicy from '../client/src/Components/ReturnPolicy.jsx';
+import SelectionList from '../client/src/Components/SelectionList.jsx';
```

Comment:
I noticed this is your `Overview.test.js` but it looks like you're testing all of your components in here?